coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -1056,7 +1056,7 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
    */
   private saveEditDirector (index, id): void {
     // get current director
-    let director = this.directors[id - 1]
+    let director = this.directors.filter(el => el.id === id)[0]
```

Comment:
A better choice for finding a single array element is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find.

Also, you should check that you found something.

Reply:
Actually, can't you just use 'index' to access the current director?

Reply:
I think Index can be used